jenkins/release-process/github-release.groovy

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env groovy
+pipeline {
+
+    agent {
+        node {
+            label 'docker-centos-7.9.2009-node'
+        }
+    }
+
+    parameters {
+        string(name: 'GIT_TAG', defaultValue: '', description: 'Image tag/Commit tag required for GitHub Release')
+        string(name: 'RELEASE_REPO', defaultValue: 'Seagate/cortx', description: 'owner/repository-name where release need to be created')
+        string(name: 'SERVICES_VERSION', defaultValue: '', description: 'Services(cortx-k8s) version on which image deployment is tested')
+        string(name: 'CHANGESET_URL', defaultValue: '', description: 'CHNAGESET.md file url.')
+        string(name: 'CORTX_RE_BRANCH', defaultValue: 'main', description: 'Branch or GitHash for GitHub Release script', trim: true)
+        string(name: 'CORTX_RE_REPO', defaultValue: 'https://github.com/Seagate/cortx-re', description: 'Repository for GitHub release script', trim: true)
+    }
+
+    environment {
+        IMAGE_TAGS = getImageTags("cortx-rgw", "2.0.0-895")
```

Comment:
Can we remove these hard-coded values?

Reply:
Removed environment variable as moved this logic to shell script